python/ganga/__init__.py

```diff
@@ -16,28 +14,33 @@
 from Ganga.Runtime import Workspace_runtime
 from Ganga.Core.GangaRepository import getRegistry
 from Ganga.GPIDev.Base.VPrinter import full_print
-from Ganga.GPIDev.Base.Proxy import implRef
+from Ganga.GPIDev.Base.Proxy import implRef, stripProxy, isProxy, addProxy
 import Ganga.GPIDev.Lib.Config
 from Ganga.Utility.feedback_report import report
 from Ganga.Runtime.gangadoc import adddoc
-from Ganga.Core.GangaThread.WorkerThreads.ThreadPoolQueueMonitor import ThreadPoolQueueMonitor
-from Ganga.Runtime import plugins
+from Ganga.Core.GangaThread.WorkerThreads import startUpQueues
+from Ganga.Core.InternalServices import ShutdownManager
+from Ganga import _gangaVersion
+from Ganga.Utility.files import expandfilename
+from os.path import expandvars, expanduser, normpath, dirname, abspath, join, isfile, islink, exists, isdir
+from inspect import currentframe, getfile
+from os import environ, listdir, pathsep
+import re
 
-logger = getLogger(modulename=True)
+logger = getLogger('ganga')
 
+logger.info("Hello from ganga")
```

Comment:
ah yes, typo on my part, was there for testing as for some reason passing 'modulename=True' didn't display anything for me